orchestration/dagster_orchestration/hca_orchestration/solids/stage_data.py

```diff
@@ -41,22 +46,14 @@ def pre_process_metadata(context: AbstractComputeExecutionContext) -> Nothing:
     """
     context.log.info("--pre_process_metadata")
 
-    context.resources.beam_runner.run(
-        "pre-process-metadata",
-        context.solid_config["input_prefix"],
-        context.solid_config["output_prefix"],
-        context
-    )
+    # not strictly required, but makes the ensuing lines a lot shorter
+    bucket_name = context.solid_config['staging_bucket_name']
+    prefix_name = context.solid_config['staging_prefix_name']
 
+    kebabified_output_prefix = re.sub(r"[^A-Za-z0-9]", "-",  prefix_name)
```

Comment:
❤️ I love that this is now as simple as it should be.

Reply:
though, hilariously, the character matching group `[^A-z0-9]` does not include underscores, for some reason no doubt related to unicode being an eldritch horror beyond human ken